hnn_core/batch_simulate.py

```diff
@@ -180,3 +200,33 @@ def _generate_param_combinations(self, param_grid, combinations=True):
             param_combinations = [dict(zip(keys, combination))
                                   for combination in zip(*values)]
         return param_combinations
+
+    def save(self, sim_results):
+        """Save simulation results to files.
+
+        Parameters
+        ----------
+        sim_results : list
+            List of simulation results.
+        """
+        if not os.path.exists(self.file_path):
+            os.makedirs(self.file_path)
```

Comment:
Just verify locally that this doesn't overwrite any files and simply adds a directory